integration/fuse/src/main/java/alluxio/fuse/file/FuseFileStream.java

```diff
@@ -105,23 +106,14 @@ public Factory(FileSystem fileSystem, AuthPolicy authPolicy) {
      */
     public FuseFileStream create(
         AlluxioURI uri, int flags, long mode) {
-      Optional<URIStatus> status = AlluxioFuseUtils.getPathStatus(mFileSystem, uri);
-      if (status.isPresent() && !status.get().isCompleted()) {
-        // Fuse.release() is async
-        // added for write-then-read and write-then-overwrite workloads
-        status = AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri);
-        if (!status.isPresent()) {
-          throw new UnimplementedRuntimeException(String.format(
-              "Failed to create fuse file stream for %s: file is being written", uri));
-        }
-      }
+      ReadWriteLock pathLock = mPathLocks.get(uri.toString());
```

Comment:
Is it possible a null returned?

Reply:
I see that, it will create one if no existing

Reply:
this will not be null